ios/Rainbow/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CodePushDeploymentKey</key>
+	<string>Pz43cwILK6DtSP38dCGPyBEofColSPrxfOgL7</string>
```

Comment:
Can we move this to the .ENV file and copy it to the xcconfig files like this https://github.com/rainbow-me/rainbow/blob/develop/scripts/postinstall.sh#L37

Then we can read it like `<string>$(CODEPUSH_DEPLOYMENT_KEY)</string>`